sql/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/streaming/ClientStreamingQuerySuite.scala

```diff
@@ -56,8 +56,8 @@ class ClientStreamingQuerySuite extends QueryTest with RemoteSparkSession with L
   test("Streaming API with windowed aggregate query") {
     // This verifies standard streaming API by starting a streaming query with windowed count.
     withSQLConf(
-      "spark.sql.shuffle.partitions" -> "1" // Avoid too many reducers.
-    ) {
+      "spark.sql.shuffle.partitions" -> "1", // Avoid too many reducers.
+      "spark.sql.streaming.stateStore.numStateStoreInstanceMetricsToReport" -> "0") {
```

Comment:
This is needed because of the below assertion:

```
lastProgress.stateOperators.head.customMetrics.keySet().asScala == Set(
            "loadedMapCacheHitCount",
            "loadedMapCacheMissCount",
            "stateOnCurrentVersionSizeBytes"))
```

Do I understand correctly? 

Technically you can even verify the metric here, but I'd be OK not to change existing tests to verify more thing while we are expected to add the test to cover this.

Reply:
Makes sense, will test for the new metric instead :+1: